src/Symfony/Component/Messenger/composer.json

```diff
@@ -19,6 +19,7 @@
         "php": ">=7.2.5",
         "psr/log": "~1.0",
         "symfony/amqp-messenger": "^5.1",
+        "symfony/beanstalkd-messenger": "^5.2",
```

Comment:
Should be removed. The idea is to add it only when needed.We do have AMQP for BC reasons.

Reply:
Oh right, removed.